metainfo_version.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/zola.metainfo.xml b/zola.metainfo.xml
+index 53bac7c0..7b6fecdb 100644
+--- a/zola.metainfo.xml
++++ b/zola.metainfo.xml
+@@ -52,8 +52,8 @@
+   <content_rating type="oars-1.1" />
+ 
+   <releases>
+-    <release version="0.15.3" date="2022-01-22">
+-      <url>https://github.com/getzola/zola/releases/tag/v0.15.3</url>
++    <release version="0.16.0" date="2022-07-16">
++      <url>https://github.com/getzola/zola/releases/tag/v0.16.0</url>
```

Comment:
Please keep both releases present. See e.g. https://github.com/flathub/com.jpexs.decompiler.flash/blob/master/com.jpexs.decompiler.flash.metainfo.xml#L65 (you don't need to add a changelog though).